### ISSUE TITLE
DBToaster regressive tests failing 

### ISSUE THREAD

**Issue by akathorn** [epfldata/squall#32]
When running `regressive_test.sh PLAN_RUNNER` from a clean Squall repo on Ubuntu 14.10 the DBToaster tests fail with this kind of error:

```
Running test 1 (0_01G_dbtoaster_hyracks) out of 31...
Exception in thread "main" java.lang.RuntimeException: Fail to prepare DBToaster dependencies 
	at ch.epfl.data.squall.utilities.StormDBToasterProvider.prepare(StormDBToasterProvider.java:63)
	at ch.epfl.data.squall.main.Main.createTopology(Main.java:148)
	at ch.epfl.data.squall.main.Main.<init>(Main.java:235)
	at ch.epfl.data.squall.main.Main.main(Main.java:157)
Caused by: java.io.FileNotFoundException: /tmp/CUSTOMER_ORDERS3042266310918904881.jar (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:215)
	at java.util.zip.ZipFile.<init>(ZipFile.java:145)
	at java.util.jar.JarFile.<init>(JarFile.java:154)
	at java.util.jar.JarFile.<init>(JarFile.java:91)
	at ch.epfl.data.squall.utilities.JarUtilities.extractJarFile(JarUtilities.java:53)
	at ch.epfl.data.squall.utilities.StormDBToasterProvider.prepare(StormDBToasterProvider.java:53)
	... 3 more
```

**Comment by akathorn** [comment]
A manual inspection of /tmp/ shows that the file exists, so maybe is a permissions issue

**Comment by khuevu** [comment]
Hi Daniel. It is actually a bug. The front_end binary I include for DBtoaster is for Mac. I will add support for Ubuntu.

**Comment by akathorn** [comment]
Solved by a24942a79f30307242581e4644fc495bdc3ded78
Still waiting to be able to run with Scala 2.10, though :)

**Comment by khuevu** [comment]
@akathorn You have trouble when running with scala 2.10 ?
